ShaggyTech/nhtsa-api-wrapper
# refactor: complete refactor, breaking changes, merge next -> main in preparation for v3 release, switch to monorepo structure

TODO:

Refactor github-actions workflows:

- build/lint/test on all branches
- implement changesets
- publish package to npm
- publish package to gh-packages
- publish docs to gh-pages via action
- ensure new version is correctly published
